Lib/glyphsLib/parser.py

```diff
@@ -105,7 +105,7 @@ def _parse(self, text, i, _parsing_unicodes=False):
                 parsed = m.group(0)
                 i += len(parsed)
                 unicode_list = m.group(1).split(",")
-                return unicode_list, i
+                return len(unicode_list)==1 and unicode_list[0] or unicode_list, i
```

Comment:
thank god black will take care of these missing spaces around binary operator from now on

Reply:
also it should be an inline/ternary if-else, right?

Reply:
i think it would be better if this always returned a list, even if it's only length one.
@belluzj ?

Reply:
I'm not sure what happens after this, I'll have to look if there's more special casing for a single unicode, maybe we use it for roundtripping correctly or something like that :(

Reply:
Always a list would make more sense.

Reply:
Yes it would. Also if you always return a list, you can probably delete those two lines that deal with a string:
https://github.com/googlei18n/glyphsLib/blob/a1a0ab59cb40ff689de2faf574259c80ee8c9b20/Lib/glyphsLib/types.py#L382-L383

Reply:
Maybe we can merge and fix this ourselves? I can commit a fix. I'm also going to look into the issue about bad axis mappings (because I made the suspicious code for axes)

Reply:
that would be great, thanks guys!